src/ui.js

```diff
@@ -26,7 +26,7 @@ export const makeTable = async () => {
       company[COMPANY_NAME_FIELD_NAME],
       company[STATUS_FIELD_NAME],
       company[CREATED_AT_FIELD_NAME],
-      company[REVENUE_YTD_FIELD_NAME],
+      new Intl.NumberFormat('fr-FR').format(company[REVENUE_YTD_FIELD_NAME]),
```

Comment:
A little bit hacky, but I think it works. The only thing I would really like to have here is a comment with an explanation of what this line of code does and why it is doing it that way. Without such a comment, a different person reading your code will have really hard time understanding what the code is doing - and more importantly why it is doing it that way.

In this situation, the comment I would write is something like 'We decided to format revenue numbers by separating them with commas. Here is the link to the original issue: link_to_issue. Conveniently enough, fr-FR locale uses commas to separate thousands - which is exactly what we needed. Therefore we decided to use this locale for formatting numbers.'